src/main/java/core/basesyntax/strategy/BalanceOperation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    private final Storage storage;
+
+    public BalanceOperation(Storage storage) {

```

Comment:
Don't begin class or method implementation with an empty line. Remove the redundant empty line at the beginning of the constructor.